.github/workflows/release-body-driver.yml

```diff
@@ -0,0 +1,121 @@
+name: Generate release body for driver
+on:
+  release:
+    types: [published]
+
+# Checks if any concurrent jobs is running for driver release CI and eventually cancel it.
+concurrency:
+  group: driver-release
+  cancel-in-progress: true
+
+jobs:
+  release-settings:
+    runs-on: ubuntu-latest
+    outputs:
+      is_latest: ${{ steps.get_settings.outputs.is_latest }} 
+    steps:
+      - name: Get latest release
```

Comment:
This is copied from the Falco one: https://github.com/falcosecurity/falco/blob/master/.github/workflows/release.yaml#L12